packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { ISparklineProps, ISparklineStyles } from './Sparkline.types';
+import type { SlotClassNames } from '../../../../react-utilities/src/index';
+import { tokens } from '@fluentui/react-theme';
+
+/**
+ * @internal
+ */
+export const hbcClassNames: SlotClassNames<ISparklineStyles> = {
```

Comment:
call these sparklineClassNames - hbc stands for horizontal bar chart

Reply:
Okay